### ISSUE TITLE
Split terminal: synchro typing mode

### ISSUE THREAD

**Issue by gsemet** [Guake/guake#1419]
A cool awaitable feature when there are several terminals is synchronized typing.
If you lock one, two or three, ... terminal in current multi-term tab and the same key will be send to every terminal (+ copy paste command, ...).

Having a small title/toolbar on each subterminal would allow to add this link icon (à-la photoshop "link layers" feature)

**Comment by gsemet** [comment]
@aichingm Wouldn't be cool to have that for 3.4 release ;) ?

**Comment by gsemet** [comment]
I think this is a must have for 3.4

**Comment by aichingm** [comment]
Hm interesting feature but I now quite sure at the moment how this would be implemented...
Guake would have to intercept all key-events on the focused terminal and emit them on the linked terminals. This could work I guess(!)...

I totally get its usefulness for typing and pasting, for example if you have 2 boxes sshed and want to execute the same instructions. But I don't know how copying text would work, which text should be copied?

I don't know if this should be a must have for the next version. Not that I think it is a bad idea, but I'm more the kind of one-baby-step-at-a-time-guy :wink:. I would say lets stabilize **split terminal**, release it and than add new features. But that's just my humble opinion.

**Comment by gsemet** [comment]
@mlouielu would you find interesting to work on this one?

**Comment by mlouielu** [comment]
Yep I've play around this, just need to figure out how to put the UI better :p
